deepmd/tf/utils/tabulate.py

```diff
@@ -126,7 +126,9 @@ def __init__(
         self.dstd = get_tensor_by_name_from_graph(
             self.graph, f"descrpt_attr{self.suffix}/t_std"
         )
-        self.ntypes = get_tensor_by_name_from_graph(self.graph, "descrpt_attr/ntypes")
+        self.ntypes = get_tensor_by_name_from_graph(
+            self.graph, f"descrpt_attr{self.suffix}/ntypes"
+        )
```

Comment:
It seems that we assumed all frozen models for compression has no `suffix` here before? @njzjz @wanghan-iapcm  
Also see https://github.com/deepmodeling/deepmd-kit/blob/devel/deepmd/tf/entrypoints/freeze.py#L109

Reply:
We assumed the top descriptor has no suffix. This is true but this is not apply to the sub descriptor in a hybrid descriptor.